examples/with-styled-components/pages/_document.js

```diff
@@ -2,20 +2,19 @@ import Document, { Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
 
 export default class MyDocument extends Document {
-  static getInitialProps ({ renderPage }) {
+  static getInitialProps({ renderPage }) {
     const sheet = new ServerStyleSheet()
-    const page = renderPage(App => props => sheet.collectStyles(<App {...props} />))
+    const page = renderPage(App => props =>
+      sheet.collectStyles(<App {...props} />)
+    )
     const styleTags = sheet.getStyleElement()
     return { ...page, styleTags }
   }
 
-  render () {
+  render() {
     return (
       <html>
-        <Head>
-          <title>My page</title>
```

Comment:
If <title> doesn't go here, where should it go?

Reply:
It should be in _app.js only.

Reply:
Great. I've just seen this --> https://github.com/zeit/next.js/blob/master/errors/no-document-title.md

But how do you make it dynamic? You just don't? Or you add it to each file in `pages`?

BTW: I'm having the same issue on the **with-react-helmet** example which is a bit nastier...

Reply:
I just add it in `<Head>` to each file in pages. I think it is the best solution for now.